docs/pages/versions/unversioned/sdk/random.md

```diff
@@ -41,10 +41,10 @@ import * as Random from 'expo-random';
 
 ## Methods
 
-### `Random.getRandomBytesAsync(byteCount)`
+### `getRandomBytes(byteCount)`
```

Comment:
Docs aren't that consistent in how we write out method description, but I think it's more common there to prefix methods with `ModuleName.` and recommend `import * as ModuleName`. Was this change intentional?

Reply:
good catch